app-frontend/src/app/pages/projects/edit/edit.html

```diff
@@ -63,6 +63,20 @@ <h5 class="sidebar-title">Editing project</h5>
           <i class="icon-info"></i>
         </div>
 
+        <!-- Annotation -->
+        <div class="list-group-item">
```

Comment:
We should probably feature flag this since it's in a part of the app that people view frequently.
A new, incomplete feature will likely confuse people.